dashboard/utils.py

```diff
@@ -0,0 +1,173 @@
+"""
```

Comment:
looks like you have a lot of forked logic around `self.financial_aid_available` being true or false. i think this should be implemented as 2 different classes with the common functionality abstracted to a parent class